lib/basedriver/image-helpers.js

```diff
@@ -2,18 +2,27 @@ import { Buffer } from 'buffer';
 import { PNG } from 'pngjs';
 import B from 'bluebird';
 
-async function cropBase64Image (base64EncodedImage, rect) {
-  const image = await createImageFromBase64(base64EncodedImage);
+/**
+ * Crop the image by given rectangle (use base64 string as input and output)
+ *
+ * @param {string} base64Image The string with base64 encoded image
+ * @param {{left: number, top: number, width: number, height: number}} rect The selected region of image
+ * @return {string} base64 encoded string of cropped image
```

Comment:
string or buffer?

Reply:
A driver's getScreenshot command returns a string, therefore it should be a string.